src/utils/MySQLDataHandler.py

```diff
@@ -393,13 +354,25 @@ def UpdateStorage(self, vcName, url, storageType, metadata, defaultMountPath):
             logger.error('Exception: %s', str(e))
             return False
 
-
     @record
```

Comment:
Please move `@record` annotation to where it belonged - above `AddVC`

Reply:
Please remove `@record` here. `init_vc_sqls` internally calls `AddVC`. It is not a direct DB call itself.